homeassistant/components/zha/core/cluster_handlers/general.py

```diff
@@ -347,26 +348,10 @@ def __init__(self, cluster: zigpy.zcl.Cluster, endpoint: Endpoint) -> None:
         super().__init__(cluster, endpoint)
         self._off_listener = None
 
-        if self.cluster.endpoint.model not in (
-            "TS011F",
-            "TS0121",
-            "TS0001",
-            "TS0002",
-            "TS0003",
-            "TS0004",
-        ):
-            return
-
-        try:
-            self.cluster.find_attribute("backlight_mode")
-        except KeyError:
-            return
-
-        self.ZCL_INIT_ATTRS = self.ZCL_INIT_ATTRS.copy()
-        self.ZCL_INIT_ATTRS["backlight_mode"] = True
-        self.ZCL_INIT_ATTRS["power_on_state"] = True
-
-        if self.cluster.endpoint.model == "TS011F":
```

Comment:
Review note:
This special case of only initializing the `child_lock` attribute for TS011F plug models also shouldn't be needed.
We can do a read once upon pairing (it's cached) to determine if it's a supported or unsupported attribute.
If it's unsupported, this is needed to not create the entity later.

The attribute is defined for all plugs, even those that don't support the functionality of the `child_lock` attribute. Seems to differ on installed firmware version (and maybe model too).